src/ChainlinkOracle.sol

```diff
@@ -51,36 +57,52 @@ contract ChainlinkOracle is IChainlinkOracle {
     /// - The vault's sample shares quoted as assets and the base feed prices don't overflow when multiplied.
     /// - The quote feed prices don't overflow when multiplied.
     /// - The vault, if set, is ERC4626-compliant.
-    /// @param vault Vault. Pass address zero to omit this parameter.
+    /// @dev The base asset is the collateral token and the quote asset is the loan token.
+    /// @param baseVault Base vault. Pass address zero to omit this parameter.
+    /// @param baseVaultConversionSample The sample amount of base vault shares used to convert to underlying.
+    /// Pass 1 if the base asset is not a vault. Should be chosen such that converting `baseVaultConversionSample` to
+    /// assets has enough precision.
     /// @param baseFeed1 First base feed. Pass address zero if the price = 1.
     /// @param baseFeed2 Second base feed. Pass address zero if the price = 1.
+    /// @param baseTokenDecimals Base token decimals.
+    /// @param quoteVault Quote vault. Pass address zero to omit this parameter.
+    /// @param quoteVaultConversionSample The sample amount of quote vault shares used to convert to underlying.
+    /// Pass 1 if the quote asset is not a vault. Should be chosen such that converting `quoteVaultConversionSample` to
+    /// assets has enough precision.
     /// @param quoteFeed1 First quote feed. Pass address zero if the price = 1.
     /// @param quoteFeed2 Second quote feed. Pass address zero if the price = 1.
-    /// @param vaultConversionSample The sample amount of vault shares used to convert to the underlying asset.
-    /// Pass 1 if the oracle does not use a vault. Should be chosen such that converting `vaultConversionSample` to
-    /// assets has enough precision.
-    /// @param baseTokenDecimals Base token decimals.
     /// @param quoteTokenDecimals Quote token decimals.
     constructor(
-        IERC4626 vault,
+        IERC4626 baseVault,
```

Comment:
why change the order, I was fine with the previous version tbh

Reply:
https://github.com/morpho-org/morpho-blue-oracles/pull/73#discussion_r1491398453